docs/who-is-using.md

```diff
@@ -22,9 +22,8 @@ title: "Who is using Verdaccio?"
 * [Gatsby](https://github.com/gatsbyjs/gatsby)
 * [Apollo GraphQL](https://github.com/apollographql)
 * [Uppy](https://github.com/transloadit/uppy)
-* [Aurelia Framework](https://github.com/aurelia)
+* [Aurelia Framework](https://github.com/aurelia/framework)
 * [bit](https://github.com/teambit/bit)
-* [Hyperledger Composer](https://github.com/hyperledger/composer)
```

Comment:
why this is gone?

Reply:
I removed it because the repo is archived 🤷🏻 

There's [Fabric](https://github.com/hyperledger/fabric) which has 11k stars, but I don't know if it uses Verdaccio.  Is there a way to tell?

Reply:
I think was renamed 
https://hyperledger.github.io/caliper/v0.2/installing-caliper/#starting-verdaccio

Reply:
Oh! Well, the [repo](https://github.com/hyperledger/caliper) only has 444 stars...do you think we should still add it?

Reply:
Also, this [composer's readme](https://github.com/hyperledger/composer) makes it sound as though Fabric was the new version...

Reply:
🙃 why not ... it is an important project for the Linux Foundation, it worth to highlight